CHANGELOG.md

```diff
@@ -10,6 +10,12 @@ Every entry has a category for which we use the following visual abbreviations:
 
 ## [Unreleased]
 
+- 🐞 The import process did not print statistics when importing events over UDP.
+  This has been fixed.  Additionally, warnings over dropped UDP messages (which
+  are to be expected due to the nature of UDP) are now gathered and then
+  presented in a readable format.
+  [#662](https://github.com/tenzir/vast/pull/662)
```

Comment:
```markdown
- 🐞 The import process did not print statistics when importing events over UDP.
   This has been fixed.  Additionally, warnings about dropped UDP packets are
   no longer shown per packet periodically reported in a readable format.
   [#662](https://github.com/tenzir/vast/pull/662)
```

Reply:
Is this missing a `but rather` between packet and periodically?

Reply:
Fixed and rebased.